linkml_runtime/utils/schemaview.py

```diff
@@ -1323,11 +1355,9 @@ def get_mappings(
                 "close": e.close_mappings,
                 "undefined": e.mappings,
             }
-        else:
-            m_dict = {}
-        if expand:
-            for k, vs in m_dict.items():
-                m_dict[k] = [self.expand_curie(v) for v in vs]
```

Comment:
this will only make changes if there is something in `m_dict`, so there is no point in applying it to `{}`